### PR TITLE
CHORE: Fixing javadoc errors

### PR DESCRIPTION
**Pull request by shimono**
Fixing javadoc errors

### CODE REVIEW

**Comment by yoh1496** [comment]
In my environment (maven 3.6.0) , there are some errors.

```
[ERROR] /data/ubuntu/dev/personium/personium-lib-es-adapter/src/main/java/io/personium/common/es/response/PersoniumSearchHit.java:88: error: tag not supported in the generated HTML version: tt
[ERROR]      * The source of the document (can be <tt>null</tt>).
[ERROR]                                           ^
[ERROR] /data/ubuntu/dev/personium/personium-lib-es-adapter/src/main/java/io/personium/common/es/response/PersoniumSearchHit.java:100: error: tag not supported in the generated HTML version: tt
[ERROR]      * The source of the document as a map (can be <tt>null</tt>).
[ERROR]                                                    ^
[ERROR] /data/ubuntu/dev/personium/personium-lib-es-adapter/src/main/java/io/personium/common/es/response/PersoniumSearchHit.java:106: error: tag not supported in the generated HTML version: tt
[ERROR]      * The source of the document as string (can be <tt>null</tt>).
[ERROR]                                                     ^
[ERROR] /data/ubuntu/dev/personium/personium-lib-es-adapter/src/main/java/io/personium/common/es/response/PersoniumSearchHit.java:112: error: tag not supported in the generated HTML version: tt
[ERROR]      * The source of the document as string (can be <tt>null</tt>).
[ERROR]                                                     ^
[ERROR] /data/ubuntu/dev/personium/personium-lib-es-adapter/src/main/java/io/personium/common/es/response/PersoniumSearchHit.java:118: error: tag not supported in the generated HTML version: tt
[ERROR]      * The source of the document as a map (can be <tt>null</tt>).
[ERROR]                                                    ^
[ERROR]
```

What version do you use?

**Comment by yoh1496** [comment]
This is solved by running javadoc in Java 8 environment.

**Review by yoh1496 — APPROVED** [review]

**Review by tochi-y — APPROVED** [review]

File: src/main/java/io/personium/common/es/response/PersoniumSearchHit.java [hunk 1]
```diff
@@ -85,8 +85,7 @@
     long getVersion();
 
     /**
-     * The source of the document (can be <tt>null</tt>). Note, its a copy of the source
-     * into a byte array, consider using {@link #sourceRef()} so there won't be a need to copy.
+     * The source of the document (can be <tt>null</tt>).
```
Comment (tochi-y) [inline src/main/java/io/personium/common/es/response/PersoniumSearchHit.java:88]:
```suggestion
     * The source of the document (can be <code>null</code>).
```

Comment (tochi-y) [inline src/main/java/io/personium/common/es/response/PersoniumSearchHit.java]:
```suggestion
     * The source of the document (can be <code>null</code>).
```

Comment (tochi-y) [inline src/main/java/io/personium/common/es/response/PersoniumSearchHit.java]:
```suggestion
     * The source of the document (can be <code>null</code>).
```

**Pull request merged**